test/test.py

```diff
@@ -0,0 +1,17 @@
+import os, sys
+from django.conf import settings
+
+test_path = os.path.join(settings.BASE_DIR, '../test/')
+sys.path.append(test_path)
+
+from crawler_test import crawler_test
+from student_test import student_test
+
```

Comment:
Are you plan to add more tests for timetables, evaluations, etc?

Reply:
Sure. I hope so.